sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -575,6 +575,7 @@ def test_window(self):
                                   [GlobalWindow()])]
     with TestPipeline() as p:
       pc = p | beam.Create(l)
+      pc = pc | beam.Map(lambda x: x)
```

Comment:
Hi @pabloem 
There is another test(`test_window_in_value`) that creates a PCollection from `WindowedValue`s.
It is used there so that typehint can be evaluated properly. Using the hack enforces it to extract the KV from the WindowedValue.

Can you please add a comment here mentioning that the hack's use here, is different from its use in the other test?

Reply:
Otherwise, LGTM :)